Visitor/Arithmetic.php

```diff
@@ -209,6 +209,17 @@ public function visit(
 
                 break;
 
+            case '#power':
+                $children[0]->accept($this, $a, $eldnah);
+
+                $acc = function ($b) use ($a, $acc) {
+                    return $acc(pow($a(), $b));
+                };
+
+                $children[1]->accept($this, $acc, $eldnah);
+                
+                break;
```

Comment:
Short answer, no. Longer, I have basically copy/pasted what is done for ```#multiplication``` and updated the operation ?